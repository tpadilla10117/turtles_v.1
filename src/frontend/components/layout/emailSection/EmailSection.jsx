/* Component Wrapper for landing page's EmailForm component: */

    import React from 'react'

    function EmailSection( {content} ) {
    return (
        <section className='emailSection-parent-container'>
            {/* <div className='emailSection-wrapper'> */}
                { content }
            {/* </div> */}

        </section>
    )
    }

    export default EmailSection;