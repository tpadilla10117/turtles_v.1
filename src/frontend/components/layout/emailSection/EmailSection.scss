.emailSection-parent-container {
    height: 500px;
    width: 100%;
    max-width: 100%;
    /* border: 10px solid red; */
}