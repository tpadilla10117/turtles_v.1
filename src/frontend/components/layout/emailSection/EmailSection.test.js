import { shallow } from 'enzyme';
import React from 'react';
import EmailSection from './EmailSection';


describe('The EmailSection.jsx component:', () => {
    let wrapper;

    beforeEach( () => {
        const mockProps = {
            bannerImg: 'text.jpeg',
            alt: 'some text'
        };

        wrapper = shallow(<EmailSection content={mockProps}/>);
        
    })

    it('renders the EmailSection.jsx component', () => {

        expect.assertions(1);
        expect(wrapper).toMatchSnapshot();
    })

   /*  it('checks for the correct props', () => {
        expect(wrapper.props().content).toEqual('<EmailForm />')
    }) */
})