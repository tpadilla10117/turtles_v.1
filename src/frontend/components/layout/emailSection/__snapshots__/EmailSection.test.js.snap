// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The EmailSection.jsx component: renders the EmailSection.jsx component 1`] = `ShallowWrapper {}`;
